src/nft/components/details/detailsV2/LandingPage.tsx

```diff
@@ -40,40 +44,56 @@ const InfoContainer = styled(ColumnCenter)`
   }
 `
 
-const InfoDetailsContainer = styled(Column)`
-  gap: 4px;
-  align-items: center;
-`
-
-const MediaContainer = styled.div`
-  width: 100%;
-  height: 100%;
+const StyledHeadlineText = styled.div`
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: ${({ theme }) => theme.textPrimary};
 
   @media screen and (min-width: ${BREAKPOINTS.sm}px) {
-    width: ${MAX_WIDTH}px;
-    height: ${MAX_WIDTH}px;
+    line-height: 44px;
+    font-size: 36px;
   }
 `
+const StyledSubheaderText = styled(ThemedText.SubHeaderSmall)`
```

Comment:
since you're setting weight, size, and height anyway doesn't really add value using the `ThemedText` over a regular div

Reply:
Oops meant to remove that